recipes/molecule-inspec/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "molecule-inspec" %}
+{% set version = "1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/molecule-inspec-{{ version }}.tar.gz
+  sha256: dd171044547ecb777e4c35c1fe02fbb294a9a2972a7c9b5ab8522fa24a9092a8
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or py < 37]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm >=1.15.0
+    - setuptools_scm_git_archive >=1.0
+  run:
+    - ansible >=2.10.3
+    - molecule >=3.2.0
+    - python
```

If (and only if!) this can be `noarch: python`:

```suggestion
    - python >=3.7
```